KoalaBot.py

```diff
@@ -124,7 +127,7 @@ def check_guild_has_ext(ctx, extension_id):
         return False
     if (not database_manager.extension_enabled(ctx.message.guild.id, extension_id)) and (not is_dpytest):
         raise PermissionError(PERMISSION_ERROR_TEXT)
-    return True
+

```

Comment:
Why was return true removed here? I haven't checked the recent usage but from the docstrings it seems pretty key
```suggestion
    return True
```